virtwho/manager/subscriptionmanager/subscriptionmanager.py

```diff
@@ -179,7 +179,8 @@ def hypervisorCheckIn(self, report, options=None):
 
         is_async = self._is_rhsm_server_async(report, connection)
         serialized_mapping = self._hypervisor_mapping(report, is_async, connection)
-        self.logger.debug("Host-to-guest mapping: %s", json.dumps(serialized_mapping, indent=4))
+        self.logger.debug("Host-to-guest mapping being sent to '%s': %s", report.config.owner,
```

Comment:
You could also use str.format() with named parameters (https://docs.python.org/2/library/stdtypes.html#str.format,